src/components/Cluster/NewCluster/ReleaseSelector/ReleaseRow.tsx

```diff
@@ -87,14 +94,17 @@ const ReleaseRow: FC<IReleaseRow> = ({
               onChange={() => selectRelease(version)}
               formFieldProps={{
                 margin: 'none',
+                tabIndex: -1,
               }}
               tabIndex={-1}
             />
           </RUMActionTarget>
         </TableCell>
-        <StyledTableCell active={active}>{version}</StyledTableCell>
-        <StyledTableCell active={active} align='center'>
-          {relativeDate(timestamp)}
+        <StyledTableCell size='small' active={active}>
```

Comment:
Added fixed sizes to these 2 columns, because previously, when you opened the list of components for a specific release, the whole table would shift. This prevents that